docs/planning/ui-views.md

```diff
@@ -196,12 +214,12 @@ interface Axis {
 }
 
 interface Label {
-  align?: CanvasTextAlign;
-  baseline?: CanvasTextBaseline;
+  align?: CanvasTextAlign; // not yet implemented
+  baseline?: CanvasTextBaseline; // not yet implemented
   color?: string;
-  fontFace?: string;
-  fontSize?: number;
-  hidden?: boolean;
+  fontFace?: string; // not yet implemented
+  fontSize?: number; // not yet implemented
+  hidden?: boolean; // not yet implemented
```

Comment:
If these aren't implemented, they oughtn't be included in the docs (unless their absence causes problems?)

Reply:
Good idea, this was copied over from the source but think removing it is the most sensible thing to do.